web/src/services/userService.ts

```diff
@@ -13,3 +13,8 @@ export const getAllUsers = () => {
 export const deleteUser = (username: string) => {
   return axios.delete(`${API_URL}/users?username=${username}`);
 };
+
+
+export const addJobOfferHelper = (data: any) => {
+  return axios.post(`${API_URL}/AddJob`, data);
```

Comment:
@navkamboj  Can you update the endpoint to /jobOffer

Reply:
@navkamboj Add employer's username to payload as employer(key) : username(value)

Reply:
Not sure if I understand it, employer(key)..?